classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+  end
```

Comment:
- [ ] @fatmahussein Great work adding the classroom class, however in order to complete the has-many relationships using the add_student method, you need to add code that ensures the student is added to the classroom itself.

```suggestion
  def add_student(student)
    @students.push(student)
     student.classroom = self
  end
```